Server/VikingVault.DataAccess/Migrations/20190813074007_DroppedProfilePages.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace VikingVault.DataAccess.Migrations
+{
+    public partial class DroppedProfilePages : Migration
```

Comment:
Is this Migration related to the registration problem?

Reply:
Maybe.. Not entirely sure :D